src/main/resources/static/buildbots.txt

```diff
@@ -0,0 +1,12 @@
+clang-x86_64-debian-fast
```

Comment:
Please make sure to document the expected input style of this file, also in the corresponding reader.

Reply:
Done! https://github.com/googleinterns/step240-2020/wiki/BuildBotSetUpReader

Reply:
Instead of inventing new file formats and parsers, we would usually just write a text proto file. 

https://github.com/protocolbuffers/protobuf

Reply:
I agree. We should add a note about that and document it in the work that needs to be done. For the work in the internship using protos instead will be out of scope.